src/Http/Http.Abstractions/src/AsParametersAttribute.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http;
+
+using System;
+
+/// <summary>
+/// Specifies that a parameter represents a structure delegate's parameter list.
```

Comment:
"structure delegate"? Do you mean "route handler delegate"?

Reply:
Actually not, It should mean something like `Specifies that a route handler delegate's parameter represents a structured parameter list`, Is that better?